### ISSUE TITLE
Tile Portal Master

### ISSUE THREAD

**Issue by liampearson96** [iChun/IssueTracker-Mods#142]
Once a portal is removed an entity like object stays behind after it has been removed. I came across this whilst doing a Laggoggles Timing report.
![2018-05-22_13 32 10](https://user-images.githubusercontent.com/11605270/40362495-94a2317c-5dc4-11e8-97a1-6cb8dd9fac87.png)

not sure if this is supposed to happen. Cheers.
Running Direwolf 1.12 modpack version 1.10

**Comment by iChun** [comment]
Does it still tick though? That shouldn't be happening.

On 22 May 2018 8:32 PM, "Liam Pearson" <notifications@github.com> wrote:

Once a portal is removed an entity like object stays behind after it has
been removed. I came across this whilst doing a Laggoggles Timing report.
[image: 2018-05-22_13 32 10]
<https://user-images.githubusercontent.com/11605270/40362495-94a2317c-5dc4-11e8-97a1-6cb8dd9fac87.png>

not sure if this is supposed to happen. Cheers.
Running Direwolf 1.12 modpack version 1.10

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/iChun/IssueTracker-Mods/issues/142>, or mute the thread

**Comment by liampearson96** [comment]
Yeah it still ticks. it only goes away when the block where the portal was is updated. which is weird as the server has been restarted several times.
